lstchain/scripts/lstchain_data_create_drs4_pedestal_file.py

```diff
@@ -66,8 +66,10 @@
 source_config = {
     "LSTEventSource": {
         "max_events":args.max_events,
+        "apply_drs4_corrections": False
```

Comment:
It is false because here we want calculate the corrections so they must not be applied.  
There is a warning because this trailets concerns a next version of ctapipe_io_lst, but since it does not damage I added it already

Reply:
I just made the release of the new `ctapipe_io_lst` version, so you could update the requirement here

Reply:
Are you sure we want to do it here?

Reply:
If you set an option that is only valid for the new version, yes. If not, don't set the option and set it in the update PR

Reply:
Ok, I take it out